setup/insert.sql

```diff
@@ -321,7 +321,19 @@ INSERT INTO `hook` (`id`, `code`, `type`, `by_module`, `block`, `native`, `activ
 (220, 'account-order.javascript-initialization', 1, 0, 0, 1, 1, 1, NOW(), NOW()),
 (221, 'account.additional', 1, 0, 1, 1, 1, 1, NOW(), NOW()),
 (222, 'account-order.product', 1, 0, 0, 1, 1, 1, NOW(), NOW()),
-(223, 'mini-cart', 1, 0, 0, 1, 1, 1, NOW(), NOW())
+(223, 'mini-cart', 1, 0, 0, 1, 1, 1, NOW(), NOW()),
+(224, 'sale.top', 1, 0, 0, 1, 1, 1, NOW(), NOW()),
```

Comment:
This is a backOffice hook ? The type must be 2 and the id must be in range 1xxx.